smaug/python/subgraph_test.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+
+""" This tests subgraphs."""
+
+import unittest
+import numpy as np
+
+from smaug.core import types_pb2
+from smaug.python.tensor import Tensor
+from smaug.python.graph import Graph
+from smaug.python.ops import math_ops
+
+parent_graph_name = "parent_graph"
+child_graph_name = "child_graph"
+grandchild_graph_name = "grandchild_graph"
+backend = "Reference"
+x = Tensor(
+    data_layout=types_pb2.N, tensor_data=np.random.rand(4).astype(np.float32))
+y = Tensor(
+    data_layout=types_pb2.N, tensor_data=np.random.rand(4).astype(np.float32))
+
+class TestUniqueName(unittest.TestCase):
+  def evaluate_nodes(self, graph, node_names):
```

Comment:
This is really just another assertion method, so I'd rename to assertGraphContains() or something along those lines.

Reply:
Done